plugins/turing/manager/experiment_manager_test.go

```diff
@@ -179,19 +177,9 @@ func TestGetExperimentRunnerConfig(t *testing.T) {
 		},
 	}
 
-	// Patch method to get passkey
+	// Patch method to get treatment service config
 	// TODO: Generate mock client and use it here instead of patching
 	em := &experimentManager{RunnerDefaults: _config.RunnerDefaults{Endpoint: "test-endpoint", Timeout: "12s"}}
-	monkey.PatchInstanceMethod(
-		reflect.TypeOf(em),
-		"GetProject",
```

Comment:
Do we need the `GetProject` function on the experiment manager anymore? Can we remove it?